index.js

```diff
@@ -55,7 +63,7 @@ class Configstore {
 			// Make sure the folder exists as it could have been deleted in the meantime
 			makeDir.sync(path.dirname(this.path), makeDirOptions);
 
-			writeFileAtomic.sync(this.path, JSON.stringify(value, null, '\t'), writeFileOptions);
```

Comment:
There should be a special handling, so if `JSON.stringify` is passed in, it should use the above argument.s